notebooks/lscp.ipynb

```diff
@@ -0,0 +1,803 @@
+     },
+     "metadata": {
+      "needs_background": "light"
+     }
+    }
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Simulate points in a network\n",
+    "\n",
+    "This function simulates points inside a network. In this case, it uses a lattice network 10x10 created by using spaghetti package. "
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "source": [
+    "def simulated_geo_points(in_data, needed=20, seed=0):\n",
```

Comment:
This function (and others used multiple times in tests and notebooks) might be a great candidate to be housed within a `locate.util.py` file. That would be more in line with reproducibility.